### DISCUSSION TITLE
我引入了springboot的包,可是并没有产生作用？

### DISCUSSION THREAD

**Discussion by minimoney12** [simple-robot/Discussions#13] [Q&A] [answered]
引入了simboot-core-spring-boot-starter 3.0.0-beta.2和simbot-component-mirai-boot  3.0.0.0-beta-M3
在配置启动类时并没有EnableSimbot注解，后尝试使用@Listener，同样爆红
是不是我哪里的配置错误了呢

**Comment by ForteScarlet** [comment, marked as answer]
查看环境中是否存在 `love.forte.simboot.spring.autoconfigure`，如果有但使用报错，参考 [这个问题](https://www.yuque.com/simpler-robot/simpler-robot-doc/rvsl9p)，如果没有，请检查依赖是否正确下载并尝试刷新、重新下载等操作

**Reply by DistanceV** [reply]
我也是这个问题，禁用了kotlin插件后就正常了
